networking/hardware_networks/about-sriov.adoc

```diff
@@ -19,49 +29,32 @@ SR-IOV Network Operator. The SR-IOV Network Operator creates and manages the
-field in each SriovNetworkNodeState Custom Resource, etc.
-
-* The SR-IOV network config daemon is a Kubernetes DaemonSet deployed on
-worker nodes when SR-IOV Operator is launched. It is responsible for
-discovering and initializing SR-IOV network devices in cluster.
-
-* The Operator webhook is a Kubernetes Dynamic Admission Controller Webhook
-that validates correctness of Operator Custom Resource and sets default values
-for fields in Operator Custom Resource that are not configured by user.
-
-* The Network resources injector is a Kubernetes Dynamic Admission Controller
-Webhook that provides functionality for patching Kubernetes Pod specifications
-with requests and limits for custom network resources such as SR-IOV VFs.
-
-* The SR-IOV network device plug-in is a Kubernetes device plug-in for
-discovering, advertising, and allocating SR-IOV network virtual function (VF)
-resources. Device plug-ins are used in Kubernetes to enable the use of limited
-resources, typically in physical devices. Device plug-ins give the Kubernetes
-scheduler awareness of resource availability, so the scheduler can schedule
-Pods on nodes with sufficient resources.
-
-* The SR-IOV CNI plug-in plumbs VF interfaces allocated from the SR-IOV device
-plug-in directly into a Pod.
+SR-IOV Network Operator::
+A controller that orchestrates discovery and management of SR-IOV network devices:
+- Watches for create, update, and delete events for Operator custom resources
+- Generates NetworkAttachmentDefinition custom resources for SR-IOV Container Network Interface (CNI)
+- Creates and updates the configuration of the SR-IOV network device plug-in
+- Creates node specific SriovNetworkNodeState custom resources
+- Updates the `spec.interfaces` field in each SriovNetworkNodeState custom resource
+
+SR-IOV network config daemon::
+A Kubernetes DaemonSet deployed on worker nodes when the SR-IOV Operator starts. The daemon is responsible for discovering and initializing SR-IOV network devices in cluster.
+
+SR-IOV Operator webhook::
+A Kubernetes dynamic admission controller webhook that validates the Operator custom resource and sets appropriate default values for unset fields.
+
+SR-IOV Network resources injector::
+A Kubernetes dynamic admission controller webhook that provides functionality for patching Kubernetes Pod specifications with requests and limits for custom network resources such as SR-IOV VFs.
+
+SR-IOV network device plug-in::
+A Kubernetes device plug-in that discovers, advertises, and allocates SR-IOV network virtual function (VF) resources. Device plug-ins are used in Kubernetes to enable the use of limited resources, typically in physical devices. Device plug-ins give the Kubernetes scheduler awareness of resource availability, so that the scheduler can schedule Pods on nodes with sufficient resources.
+
+SR-IOV CNI plug-in::
+A CNI plug-in that plumbs VF interfaces allocated from the SR-IOV device plug-in directly into a Pod.
```

Comment:
What is "plumbs" supposed to mean here? Is there a clearer word that could be used?

Reply:
@bergerhoffer, probably "attaches".